foundations/installations/installations.md

```diff
@@ -261,14 +256,81 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
 With the recent addition of being able to run a Linux terminal, the ChromeOS platform has been opened up to the ability to install native Linux applications. If you wish to use your Chromebook to complete The Odin Project, you will need to ensure you meet a couple requirements:
 
 1.  You have a supported Chromebook:
-    -   [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
-    -   [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
+    - [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
+    - [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
 2.  You can install Linux by following [these instructions](https://support.google.com/chromebook/answer/9145439?hl=en)
 
 Once you have successfully met both of these requirements, you should be able to follow along with the Linux instructions throughout the entire curriculum.
 
 </details>
 
+<details markdown="block">
+<summary class="dropDown-header">WSL2</summary>
+
+Using WSL2 is an quick and easy way to get started with using Linux, allowing you to run a Linux distribution from within Windows. WSL2 is available on Windows 10 version 2004 and higher (Build 19041 and higher) and Windows 11.
+
+<div class="lesson-note">
+<h4>WSL2 and Linux instructions</h4>
+Because WSL2 is a full-fledged Linux distribution, almost everything that the curriculum teaches about Linux is also applicable to WSL2. In future lessons, whenever there are instructions that differ by OS, you should follow the Linux instructions, unless the lesson includes WSL2-specific instructions.
+</div>
```

Comment:
```suggestion
<div class="lesson-note" markdown="1">
<h4>WSL2 and Linux instructions</h4>
Because WSL2 is a full-fledged Linux distribution, almost everything that the curriculum teaches about Linux is also applicable to WSL2. In future lessons, whenever there are instructions that differ by OS, you should follow the Linux instructions, unless the lesson includes WSL2-specific instructions.
</div>
```

`markdown="1"` should be present in note blocks (already an open issue or PR in the making to change this is styleguide)